model_api/python/openvino/model_api/models/utils.py

```diff
@@ -122,15 +122,11 @@ def load_labels(label_file):
 
 
 def resize_image(image, size, keep_aspect_ratio=False, interpolation=cv2.INTER_LINEAR):
-    if not keep_aspect_ratio:
-        resized_frame = cv2.resize(image, size, interpolation=interpolation)
-    else:
+    if keep_aspect_ratio:
         h, w = image.shape[:2]
         scale = min(size[1] / h, size[0] / w)
-        resized_frame = cv2.resize(
-            image, None, fx=scale, fy=scale, interpolation=interpolation
-        )
-    return resized_frame
+        return cv2.resize(image, None, fx=scale, fy=scale, interpolation=interpolation)
```

Comment:
We probably need to align preprocessing embedding as well.

Reply:
Removal of `parameters["resize_type"].update_default_value("crop")` affects preprocessing only. The part you left the comment under is just rephrasing, it's effectively the same

Reply:
Ok. I see